theano/sandbox/cuda/nnet.py

```diff
@@ -739,3 +739,124 @@ def c_support_code_apply(self, node, nodename):
         return ret1 + "\n" + ret2
 
 gpu_softmax_with_bias = GpuSoftmaxWithBias()
+
+
+import theano
+from theano.gof import local_optimizer
+from theano.sandbox.cuda import CudaNdarrayType
+from theano.sandbox.cuda.blocksparse import sparse_block_gemv_ss
+import theano.tensor as T
+from theano.tensor import discrete_dtypes
+
+
+class SparseBlockDot(Op):
+    """
+    This op computes the dot product of specified pieces of vectors
+    and matrices, returning pieces of vectors.
+
+    It computes something like this for each j:
+
+      o[j] = sum_over_i(dot(W[i, j], h[i])) + o[j]
+
+    The i and j are taken from the inputIdx and outputIdx lists
+    respectively.
+
+    This should not be directly called since the interface is subject
+    to change without notice.  Use the sparse_block_dot_SS() function
+    for a stable interface.
+    """
+    def __init__(self, inplace=False):
+        self.inplace = False
+
+    def make_node(self, o, W, h, inputIdx, outputIdx):
```

Comment:
Right, and it's explained [here](https://github.com/Theano/Theano/blob/master/theano/sandbox/cuda/blocksparse.py#L25). Then you should remove ```b``` for this method documentation.